docker-compose.yml

```diff
@@ -598,6 +598,24 @@ services:
         "./target/x86_64-unknown-linux-musl/debug/vector test tests/behavior/**/*.toml",
       ]
 
+  test-default:
+    build:
+      context: .
+      dockerfile: scripts/ci-docker-images/builder-x86_64-unknown-linux-musl/Dockerfile
+    environment:
+      CARGO_TERM_COLOR: always
+      TEST_LOG: debug
+      RUST_BACKTRACE: full
+    volumes:
+      - $PWD:$PWD
+      - ./target/x86_64-unknown-linux-musl/cargo/registry:/opt/rust/cargo/registry
+      - ./target/x86_64-unknown-linux-musl/cargo/git:/opt/rust/cargo/git
+      - ./target/x86_64-unknown-linux-musl/rustup/tmp:/opt/rust/rustup/tmp
+    network_mode: host
+    working_dir: $PWD
+    user: $USER
+    command: cargo test --lib --no-default-features --features default-cmake --target x86_64-unknown-linux-musl
```

Comment:
This isn't exactly `default` is it?

Reply:
It's mostly equivalent to `default` and I didn't want to use it here since `docker-compose` uses `default-cmake`. Although I'm guessing that `default` wouldn't work here. But it used in the script which gets called in CI.